### PR TITLE
Feature/iii 3302 added list of productions

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- list of productions

Screenshots:
Schermafbeelding 2020-08-13 om 09 52 54
Schermafbeelding 2020-08-13 om 09 53 10

---
Ticket: https://jira.uitdatabank.be/browse/III-3302

### CODE REVIEW

**Comment by simon-debruijn** [comment]
@corneelwille Given the layout I think we can show more than 10 productions on the screen, maybe 20?

File: i18n/fr.json [hunk 1]
```diff
@@ -43,13 +43,23 @@
...
-  "actions": {
-    "create": "toevoegen"
-  },
   "pagination": {
     "previous": "Dernier",
     "next": "Prochain"
   },
+  "productions": {
+    "create": "ajouter",
+    "delete": "suprimer",
```
Comment (brampauwelyn) [inline i18n/fr.json:52]:
```suggestion
    "delete": "supprimer",
```
You can use google translate for the translations ;)

File: pages/manage/productions/index.vue [hunk 2]
```diff
@@ -5,26 +5,167 @@
...
+            production.production_id === this.selectedProductionId,
+        );
+      },
+    },
+    async created() {
+      // get the first page of productions
+      await this.getProductions(0, this.productionsPerPage);
+    },
+    methods: {
+      async handleChangeSelectedProductionId(id) {
```
Comment (brampauwelyn) [inline pages/manage/productions/index.vue:80]: why not simply `selectProductionId`

File: pages/manage/productions/index.vue [hunk 3]
```diff
@@ -5,26 +5,167 @@
...
+      }
+
+      td {
+        width: 100%;
+      }
+    }
+
+    a {
+      font-weight: 400;
+      color: #004f94 !important;
```
Comment (brampauwelyn) [inline pages/manage/productions/index.vue:159]: Try to use as much scss variables for colors.

File: pages/manage/productions/index.vue [hunk 4]
```diff
@@ -5,26 +5,167 @@
...
+          />
+        </div>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
   import Pagination from '../../../components/pagination';
+  import LoadingSpinner from '../../../components/loading-spinner';
```
Comment (brampauwelyn) [inline pages/manage/productions/index.vue:42]: You could also do
```suggestion
  import LoadingSpinner from '@/components/loading-spinner';
```

**Comment by corneelwille** [comment]
> @corneelwille Given the layout I think we can show more than 10 productions on the screen, maybe 20?

@simon-debruijn or even make it consistent with /search/ and show 30 productions per page (= default pagination)

File: components/productions/productions.vue [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  export default {
+    components: {
+      LoadingSpinner,
+    },
+    props: {
+      productions: {
+        type: Array,
+        default: () => [],
+      },
+      isLoading: {
```
[the remark this answers is not in the document]
Comment (simon-debruijn) [reply components/productions/productions.vue:38]: @brampauwelyn 
How would we use a loader mixin when there are 2 booleans for different loading events like this: 

Schermafbeelding 2020-08-13 om 13 48 20

**Pull request merged**